magic.js

```diff
@@ -1,9 +1,9 @@
 var survivalKits = {
-  kindergarten: {poster: "images/kindergarten.jpg", title: "KinderGarten Survival Kit", price:"$40.00"}, 
-  kindergartenParents: {poster:"\\images\\kinder-parent.jpg", title:"Kindergarten Parent Survival", price:"$40.00"},
+  kindergarten: {poster: "images//school-bus.jpg", title: "KinderGarten Kit", price:"$40.00"}, 
+  kindergartenParents: {poster:"images//school-bus.jpg", title:"Kinder-Parent Survival", price:"$40.00"},
   elementarySchool: {poster:"images//school-bus.jpg", title:"Elementary Survival Kit", price:"$40.00"},
-  middleSchool: {poster:"images//middle-school.jpg", title:"Middle School Survival Kit", price:"$40.00"},
-  highSchool: {poster:"images//high-school.jpg", title:"Hight School Surivival kit", price:"$40.00"},
+  middleSchool: {poster:"images//middle-school.jpg", title:"Middle School Survival", price:"$40.00"},
```

Comment:
'Kit' needs to go in the title, but I can do that after we merge.  Spend some time with the kids!